test/e2e/upgrade/service/service.go

```diff
@@ -145,6 +145,11 @@ func (t *serviceLoadBalancerUpgradeTest) loadBalancerSetup(f *framework.Framewor
 		// - Azure is hardcoded to 15s (2 failed with 5s interval in 1.17) and is sufficient, however, it
 		// is testing against the `/healthz` path by default, switch to the `/readyz` path
 		s.Annotations[fmt.Sprintf("service.beta.kubernetes.io/port_%d_health-probe_request-path", s.Spec.Ports[0].Port)] = "/readyz"
+		// The generic annotation should apply to all platforms, but is not supported by Azure in tree.
+		// Set azure in-tree specific annotations until we move to out of tree.
+		s.Annotations["service.beta.kubernetes.io/azure-load-balancer-health-probe-protocol"] = "HTTP"
```

Comment:
not https?

Reply:
When I looked below at how the health checks on the pod port were being configured it is using HTTP for the health endpoints